src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,272 @@
+                          hidden [hidden]="!user.selectedSeats.includes('{{id}}')"
+                            xlink:href="/img/accessible-24px.svg"
+                          on="tap:AMP.setState({
+                              user: {selectedSeats: copyAndSplice(user.selectedSeats, user.selectedSeats.indexOf({{lastSelectedSeat}}), 1)}
+                          })"
+                          tabindex="0"
+                          role="button">
+                    </image>
+                  {{^available}}
+                    <image class="special-seat-image" 
+                            x="{{x}}" y="{{y}}" width="{{width}}" height="{{height}}"
+                            xlink:href="/img/accessible-unavailable-24px.svg"
+                            tabindex="0"
+                            role="button">
+                    </image>
+                  {{/available }}
+                {{/wheelchair}}
+                {{#standard }}
+                    <rect class="seat selected"
+                          hidden [hidden]="!user.selectedSeats.includes('{{id}}')"
+                          x="{{x}}" y="{{y}}" width="{{width}}" height="{{height}}"
+                          on="tap:AMP.setState({
+                              user: {selectedSeats: copyAndSplice(user.selectedSeats, user.selectedSeats.indexOf('{{id}}'), 1)}
+                          })"
+                          role="button"
+                          tabindex="0" rx="{{rx}}" ry="{{ry}}"></rect>
+                    {{#available }}
+                      <rect class="seat" 
+                          [class]= "user.selectedSeats.length ==  limitTicketsReservedSeating ? 'seat disabled-seat' : 'seat'"
+                          x="{{x}}" y="{{y}}" width="{{width}}" height="{{height}}"
+                          [hidden]="user.selectedSeats.includes('{{id}}')"
+                          on="tap:AMP.setState({
+                            lastSelectedSeat: '{{id}}',
+                              user: {selectedSeats: user.selectedSeats.concat('{{id}}')}
+                          })"
+                          role="button"
+                          tabindex="0" rx="{{rx}}" ry="{{ry}}"></rect>  
+                    {{/available }}
+                    {{^available}}
+                          <rect class="seat unavailable" 
+                          x="{{x}}" y="{{y}}" width="{{width}}" height="{{height}}"
+                          role="button"
+                          tabindex="0" rx="{{rx}}" ry="{{ry}}">   
+                          </rect>            
+                    {{/available }}
+                  {{/standard }}
+                {{/seats}}
+              </svg>
+            </amp-pan-zoom>
+          </template>
+        </amp-list>
+      </div>
+      <!-- To enable desktop users, we provide + and - buttons. We use `amp-bind` to decide whether to show the + or - button depending
+      on the state of the zooming -->
```

Comment:
Can you wrap both buttons in a div to ensure that appear in the docs

Reply:
done